MC/config/PWGLF/ini/tests/GeneratorLF_ResonancesTriggered_pp1360.C

```diff
@@ -0,0 +1,160 @@
+int External()
+{
+    std::string path{"o2sim_Kine.root"};
```

Comment:
If this should be the same, maybe we can use a symbolic link?

Reply:
I think this is same for all test scripts,
Maybe to @benedikt-voelkel ?

Reply:
Sure, if some test macros are identical, you can add symbolic links to minimise redundancy. Of course you then need to commit the links as well and Git will take care of it.

Reply:
Okay, I changed the type of them. Can we resolve this?

Reply:
ok, let's see if the tests work or if there is anything failing with the symlinks.
Let's resolve this once the CI is through or when we know that I have to do something on the CI side